src/Discord.Net.Rest/API/Common/Overwrite.cs

```diff
@@ -13,5 +13,13 @@ internal class Overwrite
         public ulong Deny { get; set; }
         [JsonProperty("allow"), Int53]
         public ulong Allow { get; set; }
+
+        public Overwrite(ulong targetId, PermissionTarget targetType, ulong allowValue, ulong denyValue)
```

Comment:
IMO this is unnecessary; these values can easily be set using brace initialization:
```cs
new Overwrite
{
   TargetId = 1234,
    // ...
    DenyValue = 1234
}
```